src/main/java/usi/si/seart/converter/GitRepoDtoToCsvConverter.java

```diff
@@ -11,10 +11,8 @@
 import usi.si.seart.dto.GitRepoDto;
 
 @AllArgsConstructor(onConstructor = @__(@Autowired))
```

Comment:
Remove the `onConstructor` argument, it's useless since this is no longer a `@Component`